backends/php-laravel/app/UseCases/Client/ClientService.php

```
@@ -149,15 +153,116 @@ public function store(ClientDto $dto): ClientStoreVo
+        ]);
+    }
+
+    /**
+     * スマホアプリからの利用開始処理を行い、アクセストークンを返します。
+     * Inactive / Suspended の場合は Active に遷移します。既に Active の場合もトークンを返します。
+     *
+     * @param ClientDto $dto クライアントDTO
+     * @return ClientStartVo 利用開始ValueObject
+     * @throws \AutoMapperPlus\Exception\UnregisteredMappingException マッピング例外
+     */
+    public function start(ClientDto $dto): ClientStartVo
```

PHPStan (2.1.54)

[error] PHPDoc tag `@throws` with type AutoMapperPlus\Exception\UnregisteredMappingException is not subtype of Throwable


(throws.notThrowable)